packages/vest/playground/index.js

```diff
@@ -0,0 +1,25 @@
+const vest = require('../dist/vest.min.js');
```

Comment:
I see why you keep them. But would consider other solutions. I even may think of installing the package as dependency to itself.

Reply:
The playground is intended for live testing during development, having the package a dependency of itself means that I will get the latest production build, but not what I am currently working on.

Reply:
I meant to `npm i .` not sure of the syntax but I know it works